res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+					<title>
+						<de>Zurück nach Hûrlýë Island</de>
+						<en>Back to Hûrlýë Island</en>
+					</title>
+					<description>
+						<de>Doch LeCruck ist inzwischen wieder zurückgesegelt, um Evelyn zu heiraten. Raymond muss die Kirche auf Hûrlýë Island stürmen, um das zu verhindern!</de>
+						<en>But LeCruck has since sailed back to marry Evelyn. Raymond must gatecrash the church on Hûrlýë Island to prevent this!</en>
+					</description>
+				</scripttemplate>
+				<scripttemplate index="11" product="2" licence_type="2" guid="a38b5c1a-ee98-49f8-83d0-06a3d27a5eaf">
+					<title>
+						<de>Das Elixier</de>
+						<en>The Elixir</en>
+					</title>
+					<description>
+						<de>Evelyn kann sich schon ganz gut selbst helfen. Und Raymond verliert auch noch das Elixier bei seiner tollpatschigen Rettungsaktion. Wie kann er nun im großen Showdown LeCruck doch noch besiegen?</de>
+						<en>Evelyn can already help herself quite well. And Raymond also loses the elixir during his clumsy rescue attempt. How can he still defeat LeCruck in the big showdown?</en>
+					</description>
+				</scripttemplate>
+			</children>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="6e1f331a-7c9d-433b-bc9f-7606ad322a5e" />
+				<job index="2" function="2" required="1" gender="2" role_guid="3ee1d44a-f3fc-47a5-aabf-25803e6e9f99" />
+				<job index="3" function="2" required="1" gender="1" role_guid="410a932c-9ceb-4080-9dbc-f82ac3c9bcd3" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="2" required="1" gender="1" />
+				<job index="6" function="2" required="1" gender="2" />
+				<job index="7" function="2" required="1" gender="1" />
+			</jobs>
+
+			<genres maingenre="1" subgenres="5,10" />
+			<episodes min="12" max="12" />
+			<blocks value="1" />
+			<price min="80000" max="130000" />
+			<potential min="59" max="95" />
+			<outcome min="71" max="95" />
+			<review min="79" max="95" />
+			<speed min="54" max="89" />
+			<studio_size value="3" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="61661455-9607-44fd-a090-c1caa8ee3205" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Dead_Winter_Dead">
+			<title>
+				<de>%dead% %winter% in %sarajevo%</de>
+				<en>%dead% %winter% in %sarajevo%</en>
+			</title>
+			<description>
+				<de>Kriegsdrama über den jungen, enthusiastischen und leicht beeindruckbaren Stevan auf der einen, und die warmherzige, aber resolute Kara auf der anderen Seite der Front in Jugoslawien, und einem Teufels-Cellisten dazwischen.</de>
+				<en>War drama about the young, enthusiastic and impressionable Stevan on one side, and the warm-hearted but resolute Kara on the other side of the front in Yugoslavia, and a devil's cellist in between.</en>
+			</description>
+
+			<variables>
+				<dead>
+					<de>Toter|Karger|Kalter</de>
```

Comment:
add Trostlos?

Reply:
synonym finders help a lot when searching for such things.

Reply:
adding some more